### ISSUE TITLE
Prototype ender not considered for PL/SQL code documentation

### ISSUE THREAD

**Issue by t-rapp** [NaturalDocs/NaturalDocs#60]
When I generate documentation with NaturalDocs v2.1 using a PL/SQL package body as input the function prototypes are rendered including the first local variable until the first semicolon `;`. It should stop at the `IS` keyword instead.

Example input file:
```sql
CREATE OR REPLACE PACKAGE BODY FOO_LIB AS

/* Function: GetFooBar */
FUNCTION GetFooBar( UserID IN INT, FooId IN INT ) RETURN BOOLEAN
IS
  v_FooType VARCHAR2(50);
  v_Result BOOLEAN;
BEGIN
  /* ... */
END;

/* Function: GetFooBarEntry */
FUNCTION GetFooBarEntry( UserID IN INT, FooId IN INT ) RETURN TFooBar IS
  v_FooEntry TFooBar;
  v_Result BOOLEAN;
BEGIN
  /* ... */
END;

/* Function: GetFooBarXML */
FUNCTION GetFooBarXML( UserID IN INT, FooId IN INT ) RETURN XMLTYPE IS
  v_Name VARCHAR2(512);
  v_Result BOOLEAN;
BEGIN
  /* ... */
END;

END FOO_LIB;
```

I have tried to update the "Languages.txt" file on project level with the following lines:
```
Alter Language: SQL

   Function Prototype Enders: ; IS AS
```
but this did not change the output.

**Comment by NaturalDocs** [comment]
SQL needed custom code to handle function prototypes so it doesn't use the enders.  It looks like I misread the Oracle documentation on this part.  Try this version: https://www.naturaldocs.org/download/temp/Natural_Docs_temp2.zip

**Comment by t-rapp** [comment]
When running the project with the test version above the function prototypes look good.
Thanks for the response!

**Comment by NaturalDocs** [comment]
This fix is in Natural Docs 2.2 Development Release 2 and higher.
